File: 프로그래머스/unrated/181851. 전국 대회 선발 고사/전국 대회 선발 고사.java
import java.util.*;
class Solution {
    public int solution(int[] rank, boolean[] attendance) {
        ArrayList <Integer> list = new ArrayList<>();
        for(int i = 0; i<rank.length; i++){
            if(attendance[i]) list.add(rank[i]);
            
        }
        Collections.sort(list);
        

        int answer = 0;
        for(int i = 0; i<rank.length; i++){
            if(list.get(0) == rank[i]) answer += i * 10000;
            else if(list.get(1) == rank[i]) answer += i * 100;
            else if(list.get(2) == rank[i]) answer += i;
        }
        
        return answer;
    }
}

File: 프로그래머스/0/181851. 전국 대회 선발 고사/전국 대회 선발 고사.js
function solution(rank, attendance) {
    var arr = [];
    
    for(let i = 0; i<rank.length; i++){
        if(attendance[i] == true) arr.push(rank[i]);
    }
    arr.sort((a, b) => a - b);
    
    return rank.indexOf(arr[0]) * 10000 + rank.indexOf(arr[1]) * 100 + rank.indexOf(arr[2]);
}